quint/io-cli-tests.md

```diff
@@ -715,6 +749,102 @@ rm out-itf-example.itf.json
 ]
 ```
 
+### Run outputs ITF with metadata
+
+<!-- !test in run itf with metadata -->
+```
+quint run --out-itf=out-itf-mbt-example.itf.json --max-steps=5 --seed=123 \
+  --invariant=totalSupplyDoesNotOverflowInv --mbt\
+  ../examples/solidity/Coin/coin.qnt
+cat out-itf-mbt-example.itf.json | jq '.states[1]'
+rm out-itf-mbt-example.itf.json
+```
+
+<!-- !test out run itf with metadata -->
+```
+{
+  "#meta": {
+    "index": 1
+  },
+  "action_taken": "mint",
+  "balances": {
+    "#map": [
+      [
+        "alice",
+        {
+          "#bigint": "0"
+        }
+      ],
+      [
+        "bob",
+        {
+          "#bigint": "111009104957917492842738448433884729279015865072041681905260877596484059410831"
+        }
+      ],
+      [
+        "charlie",
+        {
+          "#bigint": "0"
+        }
+      ],
+      [
+        "eve",
+        {
+          "#bigint": "0"
+        }
+      ],
+      [
+        "null",
+        {
+          "#bigint": "0"
+        }
+      ]
+    ]
+  },
+  "minter": "bob",
+  "nondet_picks": {
+    "amount_189": {
```

Comment:
For all names in this record, they have this annoying suffix (`_189`) for now due to unshadowing that happens before compilation. I have a fix for this in another branch (PR soon), so this will look like `amount` instead of `amount_189`, unless there are two nested `nondet amount = ...` declarations.